### PR TITLE
adding the clock element

### PR DESCRIPTION
**Pull request by jbendwick**

### CODE REVIEW

File: front_end/index.html [hunk 1]
```diff
@@ -1,91 +1,46 @@
...
-// (c) 2000-2014 ricocheting.com
-dg = new Array();
-dg[0]=new Image();dg[0].src="dg0.gif";
-dg[1]=new Image();dg[1].src="dg1.gif";
-dg[2]=new Image();dg[2].src="dg2.gif";
-dg[3]=new Image();dg[3].src="dg3.gif";
-dg[4]=new Image();[4].src="dg4.gif";
-dg[5]=new Image();[5].src="dg5.gif";
-dg[6]=new Image();[6].src="dg6.gif";
-dg[7]=new Image();[7].src="dg7.gif";
```
Comment (mikailaulmer) [inline front_end/index.html]: why is this all red??
Why are there so many conflicts?

Comment (jbendwick) [reply front_end/index.html]: these arent conflicts, they are deletions

File: front_end/style.css [hunk 2]
```diff
@@ -58,7 +58,7 @@
 	left:50%;
 	margin:-40% -1px 0 0;
 	padding:40% 1px 0;
-	background:#B96B6B;
```
Comment (mikailaulmer) [inline front_end/style.css]: I changed the colors

File: front_end/style.css [hunk 3]
```diff
@@ -2,7 +2,7 @@
 	width:180px;
 	height:180px;
 	position:relative;
-	border:8px solid #B96B6B;
+	border:8px solid #00A1FF;
```
Comment (altereagle) [inline front_end/style.css]: I like these colors

**Pull request merged**